_client/src/Ui/ViewModels/UsersManagementViewModel.cpp

```diff
@@ -79,7 +79,10 @@ UsersManagementViewModel::convertToCommonUser(const DisplayData::User &displayUs
     Common::Entities::User commonUser;
     commonUser.id       = displayUser.id.toStdString();
     commonUser.username = displayUser.username.toStdString();
-    commonUser.password = displayUser.password.toStdString();
+    commonUser.password = QCryptographicHash::hash(displayUser.password.toUtf8(),
```

Comment:
Method  `UsersManagementViewModel::convertToCommonUser` is responsible for converting ONLY
Use this hashing logic directly in View, not ViewModel.